services/web-app/components/page-tabs/page-tabs.module.scss

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright IBM Corp. 2022, 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/type' as type;
+
+.container {
+  @include breakpoint.breakpoint(md) {
+    margin-left: -(spacing.$spacing-04);
+  }
+
+  &::before {
+    position: absolute;
+    z-index: 0;
+    left: 0;
+    width: 100vw;
+    height: spacing.$spacing-09;
+    border-bottom: 1px solid theme.$layer-active-01;
+    content: '';
+  }
+}
+
+// override Carbon min height and border bottom styles
+.tab-list {
+  min-height: spacing.$spacing-09;
+
+  :global(.cds--tabs__nav-link) {
+    max-width: fit-content;
+
+    border-bottom: spacing.$spacing-02 solid transparent;
+    color: theme.$text-secondary;
+    @include type.type-style('body-short-02', true);
+
+    &:hover:not(:focus) {
+      border-bottom: spacing.$spacing-02 solid theme.$layer-active-01;
+      color: theme.$text-primary;
+    }
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  :global(.cds--tabs__nav-item--selected) {
```

Comment:
is this a carbon override? should we add an override comment like in https://github.com/carbon-design-system/carbon-platform/blob/main/services/web-app/components/catalog-sort/catalog-sort.module.scss#L79